### ISSUE TITLE
Chapter 3 : 3.12 - Exercises Problem 7

### ISSUE THREAD

**Issue by mskhan001** [fifthist/Introduction-To-Probability-Blitzstein-Solutions#23]
The solution given by you [here](https://fifthist.github.io/Introduction-To-Probability-Blitzstein-Solutions/indexsu102.html#x120-1190003.1.6) seems incorrect to me. There should be `(1-px)` term in the end to account for failing.

Refer - [math.stackexchange](https://math.stackexchange.com/questions/3845744/bob-is-playing-a-video-game-that-has-7-levels-he-starts-at-level-1-and-has-a-p)

**Comment by vmdd** [comment]
I guess you meant [this problem](https://fifthist.github.io/Introduction-To-Probability-Blitzstein-Solutions/indexsu103.html#problem-102) (or the link might have changed).

For the case of only reaching lvl 1 and no futher, we need to account for failure to go to lvl2 $(1 - p_1)$, so

$$P(X = 1) = 1 * (1 - p_1)$$

Also, it cannot be equal $1$ at $X = 1$, since that would mean it needs to be $0$ at any other point.

**Comment by RamonAraujo** [comment]
I agree with @vmdd . From the question statement, I interpret that X is the highest level that Bob can reach by playing the game. X = 1 if Bob fails to pass to level 2, which occurs with probability (1-p1).

Therefore, I think the correct answer to question 3.12 is

```math
P(X=k) = \begin{cases} 1-p_1 &, ~\text{for}~ k=1\\ p_1 p_2 \dots p_{k-1} (1-p_k) &, ~\text{for}~ k=\{ 2,3,4,5,6 \} \\ p_1 p_2 p_3 p_4 p_5 p_6 &, ~\text{for}~ k=7 \\  0 &, ~\text{otherwise}~  \end{cases}
```
